lib/middleware.js

```diff
@@ -13,6 +13,7 @@ const listTaskNames = require('./utils/summarize-tasks');
 const {
   JAMBONES_MYSQL_REFRESH_TTL
 } = require('./config');
+const { utilizeJambonzApp } = require('./apps-map');
```

Comment:
can you change the file name from apps-map to dynamic-apps

Reply:
fixed